README.md

```diff
@@ -28,32 +31,80 @@ The decision to fork the original mender-mcu-client was made to:
 (this repository).
 
-In the `mender-mcu-repository` you will find:
+In the `mender-mcu-integration` repository you will find:
 * A reference `west` workspace for Zephyr OS builds, including `CMake` and `KConfig` configurations.
 * Sample application for how to use `mender-mcu` APIs.
 * A list of boards that we have so far tested.
 
 We recommend starting there to understand how we have designed our solution and then coming back
 here for more in-depth information.
 
+### Compatibility
+| Zephyr OS version | Bootloader |
+|-------------------|------------|
+| v3.7.0            | MCUboot    |
+
+### Boards
+The reference board for `mender-mcu` is the [ESP32-S3-DevKitC](https://docs.espressif.com/projects/esp-dev-kits/en/latest/esp32s3/esp32-s3-devkitc-1/index.html#).
+
+To build the project, you need a board that supports MCUboot. The following link will filter
+the officially supported boards that also support MCUboot:
+
+* [Zephyr Project supported boards with MCU boot](https://docs.zephyrproject.org/latest/gsearch.html?q=MCUboot&check_keywords=yes&area=default#gsc.tab=0&gsc.q=MCUboot&gsc.ref=more%3Aboards&gsc.sort=)
+
+
+### Setting up a West project with `mender-mcu` as a module
+See the [Zephyr documentation](https://docs.zephyrproject.org/latest/develop/getting_started/index.html) for more information
+on getting started with Zephyr.
+
+In order to use `mender-mcu` as a [module](https://docs.zephyrproject.org/latest/develop/modules.html#modules-external-projects)
+in a [West project](https://docs.zephyrproject.org/latest/develop/west/basics.html#west-workspace), you will
+need to add `mender-mcu` to your project's west manifest:
+
+```yaml
+manifest:
+  remotes:
+  #...
+    - name: mender
+      url-base: https://github.com/mendersoftware
+  #...
+  projects:
+    - name: mender-mcu
+      remote: mender
+      revision: main
+      path: modules/mender-mcu
+      import: true
+```
+
+Then you will need to initialize a Zephyr workspace based on the manifest,
+which can be done like so:
+```
+west init workspace --manifest-url https://url/to/repository-containing-manifest
+cd workspace && west update
+```
+
+### Configuring the client
+Zephyr provides [tools](https://docs.zephyrproject.org/latest/build/kconfig/menuconfig.html) for configuring
+projects, which are used to set Kconfig options. These options are used by modules to configure their behavior.
+
+The options for configuring `mender-mcu` are found in the menuconfig under `Modules -> mender-mcu`,
+and are prefixed with `MENDER_`.
+
+You can use either `west build -t menuconfig` or `west build -t guiconfig` to open
+the configuration menu.
+
+The following option **must** be set in any configuration:
+* `MENDER_SERVER_TENANT_TOKEN` which is a token that identifies which tenant a device belongs to.
+
+You change the server from the default `https://hosted.mender.io` by changing the following option:
+* `MENDER_SERVER_HOST`
```

Comment:
Instead of just the server, I would here say something like:

The most common options to modify are the following:
* MENDER_SERVER_HOST
* intervals
* ?